src/middlewares/authenticator.js

```diff
@@ -45,6 +45,7 @@ module.exports = async function (req, res, next) {
 	try {
 		let roleValidation = false
 		let decodedToken
+		const isBearerRequired = process.env.IS_AUTH_TOKEN_BEARER === 'true'
```

Comment:
@priyanka-TL Do we need this since anyways default value will be set right?

Reply:
@aks30  based on this key, we written the logic to get the token, like remove bearer if IS_AUTH_TOKEN_BEARER is false.